plugin-k8s/Cargo.toml

```diff
@@ -13,3 +13,5 @@ reqwest = { version = "0.12", features = ["json"] }
 tokio = "1.37.0"
 serde = { version = "1.0.198", features = ["derive"] }
 humantime-serde = "1.1.1"
+inotify = "0.10.2"
+lazy_static = "1.4"
```

Comment:
```suggestion

```
You'll be able to remove this dependency once the code is cleaner (see comments on the lazy_static)